package.json

```diff
@@ -58,24 +58,24 @@
     "@nomicfoundation/hardhat-chai-matchers": "^1.0.4",
     "@nomiclabs/hardhat-ethers": "^2.0.6",
     "@nomiclabs/hardhat-etherscan": "^3.1.0",
-    "@openzeppelin/contracts": "^4.7.3",
-    "@openzeppelin/contracts-upgradeable": "^4.7.3",
+    "@openzeppelin/contracts": "4.7",
+    "@openzeppelin/contracts-upgradeable": "4.7",
     "@openzeppelin/hardhat-upgrades": "^1.19.0",
     "@tableland/local": "^0.0.3",
     "@tableland/sdk": "^3.1.4",
```

Comment:
Can we change these lines to: 
```
"@tableland/local": "^1.0.0-pre.1",
"@tableland/sdk": "^4.0.0-pre.2",
```
?

Reply:
yes of course, good point!

Reply:
oof, circular deps are tricky. at least just dev deps on this end.

Reply:
oh wait, the more recent update for @tabeland/local on npm is 0.0.5-pre.2, which is a month old. do we wanna publish the 1.0.0-pre.1 first and then do the update here?

Reply:
also, i'll do another pre-release on the sdk so we can use that